README.md

```diff
@@ -60,7 +60,7 @@ submit new unit tests for old code. Unit tests can be compiled and run (assuming
 
 Every pull request is built for both Windows and Linux on a dedicated server,
 and unit and sanity tests are automatically run. The binaries produced may be
-used for manual QA testing — a link to them will appear in a comment on the
+used for manual QA testing -- a link to them will appear in a comment on the
```

Comment:
This file is used on servers as well. You "cat", "less" or "view" it. A lot of servers are set on ASCII-only in order to be less buggy while editing configuration files. Call it UX for sysadmin, but all the servers I work with display those chars as spurious chars. If you browse projects as gcc, libc and a lot of others, you find that they are in 2015 too, but they care about the "sysadmin UX" more than we do.